OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -197,6 +197,15 @@ public void DisplayMessage(string text, string prefix = "Mission", Color? color
 			TextNotificationsManager.AddMissionLine(prefix, text, c);
 		}
 
+		[Desc("Display a text message to the player.")]
+		public void DisplayMessageToPlayer(Player player, string text, string prefix = "Mission", Color? color = null)
+		{
+			if (world.LocalPlayer != player)
+				return;
+
+			DisplayMessage(text, prefix, color);
```

Comment:
Couldn't the translation be inside this function?

Reply:
Could, but what about non-translateable strings? Also keeping it localised in the script and not hidden behind the API makes it easier to discover.